Lib/concurrent/futures/process.py

```diff
@@ -373,7 +373,7 @@ def wait_result_broken_or_wakeup(self):
         assert not self.thread_wakeup._closed
         wakeup_reader = self.thread_wakeup._reader
         readers = [result_reader, wakeup_reader]
-        worker_sentinels = [p.sentinel for p in self.processes.values()]
+        worker_sentinels = [p.sentinel for p in self.processes.copy().values()]
```

Comment:
```suggestion
        worker_sentinels = [p.sentinel for p in list(self.processes.values())]
```
There is no need to copy the entire dict if only values are needed.
The result is almost the same but proposed version is more idiomatic, stdlib uses this approach in other modules.